### PR TITLE
Hude/better tests 2

### PR DESCRIPTION
**Pull request by clothic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "prettier:assemblyscript": "prettier --write ./assembly/**/*.ts",
     "prettier:ts": "prettier --write ./src.ts/**/*.ts ./tests/*.ts",
     "prettier": "npm-run-all prettier:assemblyscript",
-    "test": "ts-mocha --timeout 0 ./tests/**.spec.ts",
-    "test:view": "ts-mocha --timeout 0 ./tests/view.spec.ts",
-    "test:release": "ts-mocha --timeout 0 ./tests/release.spec.ts",
+    "test": "ts-mocha --timeout 0 --reporter spec ./tests/**.spec.ts",
```
Comment (clothic) [inline package.json:17]: what does reporter do?

File: block_validation_utils.ts
Comment (clothic) [inline block_validation_utils.ts:1]: where is this file used? maybe put it into one of the directories like src.ts/utils or something. Add a package.json script to use this file

**Review by clothic — COMMENTED** [review]
Fix comments and merge LTGM

**Comment by clothic** [comment]
also first test is failing

File: package.json (hunk 1 above)
Comment (hudeintres) [reply package.json:17]: needed for it to give a summary at the end to tell you what tests passed and failed

File: block_validation_utils.ts
Comment (hudeintres) [reply block_validation_utils.ts:1]: not used yet, but may be used in the future. I was originally gonna validate blocks but it's kinda unnecessary work cause we know the main btc blockchain is going to have correct state

**Comment by hudeintres** [comment]
You need to run `yarn test:release` everything passes for me

**Pull request merged**